docs/UserGuide.md

```diff
@@ -649,21 +648,24 @@ SUCCESS: Medicine has been deleted
 
 ## Search
 
-To search for a medication in the library, you can type 'search', followed by the field you want to search for and the keyword 
-`keyword`.
+MediTracker has a built-in medication library where you can search for common medications. This library contains 
+information on the medication name, the illness that the medication is used for, and the side effects of the medication.
+
+To search for a medication in the library, you can type 'search', followed by the field you want to search for and the 
+keyword.
 You have to include at least one of the following flags: `-n`, `-i`, `-s`, `-a` to specify the field you want to 
 search for.
 
-
 ```
 Usage:
 	search [-n name] [-i illness] [-s sideEffects] [-a allFields] [-h]
 Options:
-	-n name                 Name of medication
-	-i illness              Illness that the medication is used for
-	-s sideEffects          Side effects of the medication
-	-a allFields            Search all fields
-	-h                      Prints this help message
+
```

Comment:
You could remove this additional space to follow the format we have.

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/238/commits/0dc2d4a8f35532244568b60b7666f4e8aa6308ce